scripts/run_simulation.py

```diff
@@ -0,0 +1,121 @@
+import sys
+import os
+from io import BytesIO
+
+import yaml
+
+import numpy as np
+from mpi4py import MPI
+
+from rexfw.convenience import setup_default_re_master
+from rexfw.convenience import setup_default_replica
+from rexfw.slaves import Slave
+from rexfw.samplers.rwmc import RWMCSampler
+from rexfw.pdfs.normal import Normal
+from rexfw.communicators.mpi import MPICommunicator
+
+# gross hack to get around the issue of dependency on a module in a
+# subdirectory of a Git repositry
+# (https://github.com/python-poetry/poetry/pull/2242)
+sys.path.append(os.getenv('RESAAS_COMMON_PATH',
```

Comment:
IMO we can drop this approach in favor of the packaging options discussed above.

Reply:
Removed with addition of `resaas` submodule.